content/authorization/application-registration-prerequisites.md

```diff
@@ -0,0 +1,211 @@
+       https://mxtoolbox.com/txtlookup.aspx
+       https://gf.dev/dns-lookup
+</p>
+</details>
+<details><summary>Inspect your TLS Certificate</summary>
+
+TLS certificates are used to protect both the end users' information while it's in transfer, and to authenticate the website's organization identity to ensure users are interacting with legitimate website owners.
+
+To follow the security best practices you should utilize TLS implementations that support current IETF best practices for TLS (please refer to [this documentation](http://fhir.cerner.com/authorization/tls-guidance/) for more details). Cerner RECOMMENDS you utilize public testing tools, such as [Qualsys SSL Labs](http://ssllabs.com/), to verify proper functioning of your https implementation.
+
+Please follow one of below approaches to check TLS Certificates for your application:
+
+   1. Using OpenSSL Mac/Linux:
+        https://www.feistyduck.com/library/openssl-cookbook/online/ch-testing-with-openssl.html
+
+   1. Using the following online resources:
+       * https://www.ssllabs.com/ssltest/index.html
+       * https://ssltools.digicert.com/checker/
+       * https://www.thesslstore.com/ssltools/ssl-checker.php
+
+What you should pay attention to:
+
+   1. Supported TLS Certificates types:
+       * Domain Validated (DV),
+       * Organization Validated (OV),
+       * Extended Validation (EV).
+   1. Supported TLS protocols:
+       * TLS 1.2
+       * TLS 1.3
+   1. The Certificate Revoked/not Revoked status
+   1. The Certificate “Valid from” date
+   1. The Certificate expiration date
+   1. The Certificate issued by one of the public [Certificate Authorities](https://wiki.mozilla.org/CA/Included_Certificates).
+
+Example
+
+Using online resource https://www.ssllabs.com TLS certificate result may be as on example below:
+![Example Image](cert_example.png)
+</p>
+</details>
+
+# Information Displayed to Users
+### Application / Business Details
+
+The following table explains how following the security practices above result in users getting more information about your application and the business/organization offering it.  The table shows what information Cerner displays to end users, where it's sourced from, and under what conditions it's used.
+
+|Information | Example | Source | Conditions |
+| ------------- | ------------- |------------- |------------- |
+|  Application Name  | Example Health App  | Cerner CODE Portal - Application Registration  | 	Application name is only displayed to users if a legal entity can be associated with the application's website via the use of a valid, trusted EV or OV certificate.  |
+| Organization Name  | Example Health Corporation  | EV or OV TLS Certificate  | rganization Name	Example Health Corporation	EV or OV TLS Certificate	Organization name is only displayed to users if a legal entity can be associated with the application's website via the use of a valid, trusted EV or OV certificate.   |
```

Comment:
I think the "Conditions" column in this row got "rganization Name	Example Health Corporation	EV or OV TLS Certificate" accidentally copy pasted in?

Reply:
FIxed.